.github/workflows/go.yml

```diff
@@ -62,8 +37,17 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v4
 
+    - name: Build
+      run: go build -v ./...
+
+    - name: Unit Tests
+      run: go test -v -short ./...
+
     - name: Integration Tests
```

Comment:
thoughts on not running an integration test for the entire matrix, just most recent golang version for each OS? I'd like to include up to `1.21` in CI and running integ tests for `3 OS * 4 version * 10m` isn't free.

Reply:
I think that'll be a safe bet.